From 4d192a0507c9536933216988d8c3d3649ee9c771 Mon Sep 17 00:00:00 2001
From: Mel Gorman <mgorman@suse.de>
Date: Fri, 5 Jan 2018 10:07:54 +0000
Subject: [PATCH] cpufreq, intel_pstate: Update setpoint for unspecified ACPI
 profile

References: bnc#1068680 Update schedutil and intel_pstate to default to load-based policy
Patch-mainline: No, upstream favours power consumption over performance

At least some HP Proliant machines do not set a preferred profile at all
and the setpoint is not updated. This leads to a case whereby two very
similar HP Proliant machines have a different performance profile due
to different setpoints being used. This patch updates the setpoint for
such machines.

Signed-off-by: Mel Gorman <mgorman@suse.de>
---
 drivers/cpufreq/intel_pstate.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/cpufreq/intel_pstate.c b/drivers/cpufreq/intel_pstate.c
index c10149c656a5..855c8784742a 100644
--- a/drivers/cpufreq/intel_pstate.c
+++ b/drivers/cpufreq/intel_pstate.c
@@ -2614,8 +2614,8 @@ static int __init intel_pstate_init(void)
 			profile = "Performance Server";
 			break;
 		default:
-			if (server_policy)
-				profile = "Server";
+			profile = "Unspecified SLE Server";
+			break;
 		};
 
 		if (profile) {
